LICENSE.md

```diff
@@ -262,6 +262,7 @@ _This file was generated by the `bin/docs.py` script on 2020-03-30._
 - **Quattroshapes Point Gazetteer**: source [data](http://www.quattroshapes.com/) and [license](https://github.com/foursquare/quattroshapes/blob/master/LICENSE.md). This source includes `CC-BY` compatible data from the following organizations:
   - **Global**: GeoPlanet
   - **Global**: GeoNames
+- **Romanian National Agency for Cadastre and Land Registration (ANCPI)**: source [data](https://geoportal.ancpi.ro/geoportal/catalog/download/download.page) and [license](https://geoportal.ancpi.ro/portal/sharing/rest/content/items/8077dd16e957477d9cc9b457f5f17cac/data)
```

Comment:
The data link 503s for me, but I remember it working before.

Otherwise LGTM.

Reply:
This also works: http://geoportal.ancpi.ro/portal/apps/webappviewer/index.html?id=faeba2d173374445b1f13512bd477bb2

I'll update this link - it looks like the agency is migrating data services, per https://github.com/whosonfirst-data/whosonfirst-data/issues/1741#issue-522999358